lib/licensed/source/npm.rb

```diff
@@ -34,6 +34,7 @@ def dependencies
       def packages
         root_dependencies = JSON.parse(package_metadata_command)["dependencies"]
         recursive_dependencies(root_dependencies).each_with_object({}) do |(name, results), hsh|
+          next if @config.ignored?(hsh)
```

Comment:
I don't think this will work the way you're expecting.  `hsh` in this case is an accumulator of `{ <name> => { <package data> } }` pairs, and doesn't have `@config.ignored?`s expected structure of `{ "name" => <name>, "type" => <type> }`

Reply:
Ahh that makes sense! Thanks for the insight there. I was able to structure this in the same manner as you outlined for Go and it seems to be excluding as I expect it to. The changes I made in [Licensed::Command::List](https://github.com/github/licensed/blob/24b60232243f5773a16318ab51fe5f713e25c81e/lib/licensed/command/list.rb) did not seem to be necessary.